app/models.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Actor:
+    id: int
```

Comment:
Ensure that the `id` attribute is managed correctly when creating `Actor` instances. Typically, the database should handle the generation of unique IDs, so you might want to consider making `id` optional or handling it separately.